examples/hacker-news/package.json

```diff
@@ -53,14 +54,11 @@
     "react-router-dom": "^4.2.2",
     "redux": "^3.7.2",
     "redux-saga": "^0.16.0",
-    "resolve-bus-memory": "0.7.4",
-    "resolve-command": "0.7.4",
-    "resolve-es": "0.7.4",
-    "resolve-query": "0.7.4",
-    "resolve-readmodel-memory": "0.7.4",
     "resolve-redux": "0.7.4",
     "resolve-scripts": "0.7.4",
+    "resolve-es": "0.7.4",
     "resolve-storage-lite": "0.7.4",
+    "resolve-bus-memory": "0.7.4",
```

Comment:
Does new `resolve-scripts` packages-based architecture require implicit `bus` and `es` packges enumeration in appropriate `package.json` ?